samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/print/PDFAssessmentBean.java

```diff
@@ -667,6 +671,7 @@ else if (item.getItemData().getTypeId().equals(TypeIfc.ESSAY_QUESTION)) {
 					item.getItemData().getTypeId().equals(TypeIfc.TRUE_FALSE) ||
 					item.getItemData().getTypeId().equals(TypeIfc.FILL_IN_BLANK) ||
 					item.getItemData().getTypeId().equals(TypeIfc.FILL_IN_NUMERIC) ||
+					item.getItemData().getTypeId().equals(TypeIfc.CALCULATED_QUESTION) ||
```

Comment:
And here too I guess. Even though it will look different than the surrounding code, if we're changing the line we might as well make it null safe.